app/controllers/drafts_controller.rb

```diff
@@ -47,7 +47,13 @@ def language_id
 
   def edit
     translation = load_translation
-    translation.update_draft(data_attrs)
+
+    # Note that this endpoint isn't used for anything other than publishing to s3. Eventually once the new client switches
+    # to the new way of publishing, we'll delete this file
+    # see: https://jira.cru.org/browse/GT-2182
+    do_publishing = data_attrs[:is_published]
+
+    translation.push_published_to_s3
```

Comment:
I don't think we ever call this endpoint with anything other than `is_published: true`, but to be safe we probably should protect the call to `translation.push_published_to_s3` with `if do_publishing`